### DISCUSSION TITLE
FetchError: unable to get local issuer certificate

### DISCUSSION THREAD

**Discussion by Dima-work-account** [vercel/community#7421] [Account Issue] [answered]
### Question

I try to log in via Vercel CLI but get an error. I get a page with success confirmation (right email address) and this error in the console after that.

FetchError: request to https://api.vercel.com/registration/verify?XXXXXXXXXXXXXXXXXXXXX, reason: unable to get local issuer certificate
    at ClientRequest.<anonymous> (XXXXXXXXXXX\AppData\Local\pnpm\global\5\.pnpm\vercel@34.3.1\node_modules\vercel\dist\index.js:60310:18)
    at ClientRequest.emit (node:events:519:28)
    at ClientRequest.emit (node:domain:488:12)
    at TLSSocket.socketErrorListener (node:_http_client:500:9)
    at TLSSocket.emit (node:events:519:28)
    at TLSSocket.emit (node:domain:488:12)
    at emitErrorNT (node:internal/streams/destroy:169:8)
    at emitErrorCloseNT (node:internal/streams/destroy:128:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)

Win 11 -> Firefox -> Vercel Next.js learn app

What can I do?

**Comment by pawlean** [comment]
Hi @Gentle-Cannibal!

The error you're encountering suggests that there's an issue with the SSL/TLS certificate verification on your system. Perhaps this is related to a missing or misconfigured certificate authority (CA) bundle? Unsure if this is a Vercel issue, but here are some useful documentation: 

   - [Vercel CLI Documentation](https://vercel.com/docs/cli)
   - [Node.js SSL/TLS Options](https://nodejs.org/api/cli.html#cli_node_extra_ca_certsfile)
   - [npm config - cafile](https://docs.npmjs.com/cli/v7/configuring-npm/npmrc#cafile)
   - [What is an SSL/TLS Certificate?](https://www.digicert.com/what-is-ssl-tls-certificate)
   - [Node.js SSL/TLS Overview](https://nodejs.org/en/docs/guides/secure-development/#https)
   - [Node.js Help](https://github.com/nodejs/help)
   - [CA Certificates from curl.se](https://curl.se/docs/caextract.html)

**Comment by dprothero** [comment, marked as answer]
For us, this was caused by a network security tool that is installed by our IT department. Turning it off fixes the issue, but it would be good to know more about what the Vercel CLI is trying to do beyond making a simple https request.

[a comment by an automated account is not shown]
